src/commons/s3.rs

```diff
@@ -4,9 +4,12 @@
 //! Operator CRDs are expected to use the [S3BucketDef] as an entry point to this module
 //! and obtain an [InlinedS3BucketSpec] by calling [`S3BucketDef::resolve`].
 //!
-use crate::commons::{authentication::tls::Tls, secret_class::SecretClassVolume};
-use crate::error;
-use crate::{client::Client, error::OperatorResult};
+use crate::{
```

Comment:
Good to see these imports grouped, but they need to go under the third-party imports :)

Reply:
Fixed in 75d7cea